BedrockServer.cpp

```diff
@@ -1864,6 +1891,17 @@ void BedrockServer::_control(unique_ptr<BedrockCommand>& command) {
             SQLite::enableTrace.store(command->request.test("enable"));
             response["newValue"] = SQLite::enableTrace ? "true" : "false";
         }
+    } else if (SIEquals(command->request.methodLine, "EnableEscalateOverHTTP")) {
```

Comment:
NAB: I don't love "enable" command names that are actually "toggle" commands, but I won't hold up the PR over it.